cmd/ipfs-cluster-ctl/main.go

```diff
@@ -174,7 +166,26 @@ requires authorization. implies --https, which you can disable with --force-http
 			checkErr("", errors.New("unsupported encoding"))
 		}
 
-		globalClient, err = client.NewDefaultClient(cfg)
+		var configs []*client.Config
+		var err error
+		for _, addr := range c.StringSlice("host") {
+			multiaddr, err := ma.NewMultiaddr(addr)
+			checkErr("parsing host multiaddress", err)
+
+			cfgs, err := cfg.AsTemplateForResolvedAddress(multiaddr)
```

Comment:
This would receive a context. `c` is a context already, I think it can be passed directly.

Reply:
I think c (`c *cli.Context`) is a different context struct provided by the library. I can use the `ctx` variable created as soon as `main` function begins.

Reply:
You're right, I was looking at urfave/cli/v2 documentation where it actually implements `context.Context`. Ok, use that global context for the moment.